local/local.go

```diff
@@ -722,7 +722,11 @@ func untar(r io.Reader, dest string) error {
 			if err != nil {
 				return err
 			}
-			if _, err := io.Copy(fh, tr); err != nil {
+			if _, err := io.CopyN(fh, tr, 1024);
```

Comment:
As you can see [here](https://github.com/buildpacks/imgutil/pull/113/checks?check_run_id=2570564280) and [here](https://github.com/buildpacks/imgutil/pull/113/checks?check_run_id=2570564240), the tests are failing because of the following typo:
```suggestion
			if _, err := io.CopyN(fh, tr, 1024) {
```

Note: You can get to the failing tests by clicking on "Details" next to the failing tests at the bottom of the [PR's conversation tab](https://github.com/buildpacks/imgutil/pull/113).